components/javascript/src/main/java/org/datacleaner/beans/script/JavaScriptTransformer.java

```diff
@@ -52,9 +52,7 @@
 	private static final Logger logger = LoggerFactory
 			.getLogger(JavaScriptTransformer.class);
 
-	public static enum ReturnType {
-		STRING, NUMBER, BOOLEAN;
-	}
+	public enum ReturnType {STRING, NUMBER, BOOLEAN}
```

Comment:
Maybe it's wrong to change the convention right now at least :-) And we do have a habit of also letting our enums implement interfaces such as ```HasName``` or adding other methods to them.